backend/tracim_backend/views/base.py

```diff
@@ -1,13 +1,25 @@
+from http import HTTPStatus
+import json
+
 from pyramid.config import Configurator
 
 from tracim_backend import PageNotFound
 from tracim_backend import TracimRequest
 from tracim_backend.views.controllers import Controller
+from tracim_backend.views.errors import ErrorSchema
 
 
 class BaseController(Controller):
     def not_found_view(self, context, request: TracimRequest):
-        raise PageNotFound("{} is not a valid path".format(request.path)) from context
+        # FIXME  - G.M - 2022-02-24 - Explicit error return as the global handle_exception does
```

Comment:
Don't forget to create a GitHub issue for this FIXME.

Reply:
Is this issue created?

Reply:
yes : #5487

Reply:
```suggestion
        # FIXME  - G.M - 2022-02-24 - Explicit error return as the global handle_exception does, 
        # see https://github.com/tracim/tracim/issues/5487
```